go/oasis-node/cmd/debug/dumpdb/dumpdb.go

```diff
@@ -104,10 +104,11 @@ func doDumpDB(cmd *cobra.Command, args []string) {
 	ldb, _, stateRoot, err := abci.InitStateStorage(
 		ctx,
 		&abci.ApplicationConfig{
-			DataDir:           filepath.Join(dataDir, tendermintCommon.StateDir),
-			StorageBackend:    storageDB.BackendNameBadgerDB, // No other backend for now.
-			MemoryOnlyStorage: false,
-			ReadOnlyStorage:   viper.GetBool(cfgDumpReadOnlyDB),
+			DataDir:            filepath.Join(dataDir, tendermintCommon.StateDir),
+			StorageBackend:     storageDB.BackendNameBadgerDB, // No other backend for now.
+			MemoryOnlyStorage:  false,
+			ReadOnlyStorage:    viper.GetBool(cfgDumpReadOnlyDB),
+			CheckpointInterval: 1 * time.Minute,
```

Comment:
Doesn't make sense to enable checkpointing here? Disable checkpoints in this case.